trillian.proto

```diff
@@ -194,20 +194,15 @@ message SignedLogRoot {
   int64 tree_revision = 6;
 }
 
-message MapperMetadata {
-  bytes source_log_id = 1;
-  int64 highest_fully_completed_seq = 2;
-  int64 highest_partially_completed_seq = 3;
- }
-
 // SignedMapRoot represents a commitment by a Map to a particular tree.
 message SignedMapRoot {
   int64 timestamp_nanos = 1;
   bytes root_hash = 2;
-  MapperMetadata metadata = 3;
+  reserved 3;
 	// TODO(al): define serialized format for the signature scheme.
   sigpb.DigitallySigned signature = 4;
 
   int64 map_id = 5;
   int64 map_revision = 6;
+  google.protobuf.Any metadata = 7;
```

Comment:
Please document. In particular, what types allowed/expected to be in the Any?

Reply:
The intention is that any protocol message could be stored here, via ptypes Marshaling.
In reality the Trillian storage in use will probably impose a size limit on that, but I'm not sure
I can/should document that here other than saying something general about it.

Reply:
My questions were mostly around purpose, lifecycle and content restrictions (if any). You've made those clear.